spec/support/controller_spec_helper.rb

```diff
@@ -7,16 +7,6 @@ def assigns(key = nil)
     end
   end
 
-  def set_user_privileges(user = FactoryGirl.create(:user_with_group))
-    allow(User).to receive(:server_timezone).and_return("UTC")
-    allow_any_instance_of(described_class).to receive(:set_user_time_zone)
-
-    # TODO: remove these stubs
-    allow(controller).to receive(:check_privileges).and_return(true)
-    login_as user
-    allow_any_instance_of(User).to receive(:role_allows?).and_return(true)
```

Comment:
We could potentially do stubbing of specific features (that's why the API here is so explicit, so if we run into a reason to do so someday...), but that's not something we're interested in doing quite yet.